### DISCUSSION TITLE
Will there be protections for Marketplace Worlds?

### DISCUSSION THREAD

**Discussion by ihategravel2** [Mojang/minecraft-editor#113] [Q&A] [answered]
Will protections/restrictions for Marketplace Worlds be implemented such as restriction on Structures and other aspects?

**Comment by minecraftchloe** [comment, marked as answer]
Very good question. Nothing more than what is already in place today. If you download and then import a Marketplace map into the Editor, it's effectively the same thing as importing as though you're going to play, except it converts it to an .mcproject and you get the Editor interface to do Editor _thingz_.

**Reply by Landy005** [reply]
Wouldn't this be a way around the security, if you can convert it to a mcproject and then save it as a new world which you can then export and bypass the encryption? Or would it still be encrypted.

**Reply by ihategravel2** [reply]
It's still encrypted
